### ISSUE TITLE
4.4.3 still hangs the dock when dragging icons from it.

### ISSUE THREAD

**Issue by etrahretep** [w0lfschild/cDock-Issue-Tracker#294]
**Describe the bug**
When dragging an icon from the dock or moving one from a folder on the dock, the dock hangs repeatedly
Have communicated a few times earlier about this issue. Renders the program totally unusable.

**To Reproduce**
Steps to reproduce the behavior (after each restart/login):
1. open folder in dock (sometimes it restarts the dock)
<img width="553" alt="Schermafbeelding 2020-11-07 om 16 22 02 kopie" src="https://user-images.githubusercontent.com/42773309/98445215-b7b4ab00-2116-11eb-99bf-854a56e771c6.png">
2. close and try dragging an icon from the dock
<img width="260" alt="Schermafbeelding 2020-11-07 om 16 18 23" src="https://user-images.githubusercontent.com/42773309/98445094-1594c300-2116-11eb-9766-9e90665e2aee.png">
3. Dock hangs.
<img width="344" alt="Schermafbeelding 2020-11-07 om 16 19 18" src="https://user-images.githubusercontent.com/42773309/98445122-4117ad80-2116-11eb-96be-d8c6a1da9a84.png">
4. Manual restart dock to fix it until the next sequence

**Desktop (please complete the following information):**
 - OS: macOS 11.0.1
11.0.1 bèta (20B5022a)

**Additional context**
Tried 3 times to get a refund through Paddle but they redirected me back to the developer.
Not happy with this service at all.
cDock is unusable on the latest version of Big Sur beta.
Simply put: Refund the purchase and let’s forget about it.

**Comment by w0lfschild** [comment]
Fixed in 4.5.0
